pulp_ansible/tests/functional/utils.py

```diff
@@ -32,6 +32,7 @@
 
 
 configuration = Configuration()
+configuration.host = config.get_config().get_base_url()
 configuration.username = "admin"
```

Comment:
I see the pulp smash config also contains the username and password can this PR also have those configured from the settings too?

If we do that, with multiple configuration points coming from the smash config what about making a new function that will effectively return the fully configured `Configuration()` object that all bindings could be instantiated with?

What do you think?

Reply:
sounds good, I will do that

Reply:
@fabricio-aguiar great, I can re-review if you ping me when it's ready.

Reply:
done!